process_report/tests/e2e/test_e2e_pipeline.py

```diff
@@ -178,24 +181,35 @@ def _validate_outputs(workspace: Path) -> None:
         AssertionError: If any expected output file is missing, empty, or invalid.
         pytest.fail: If CSV files cannot be read or parsed.
     """
-    logger = logging.getLogger(__name__)
     logger.info(f"Validating pipeline outputs in: {workspace}")
 
-    # Validate CSV files
+    # Basic existence and structure validation catches pipeline execution failures
```

Comment:
I would remove this comment. I would say your validation code here has explained itself.